### PR TITLE
review store in progress

### PR DESCRIPTION
**Pull request by iankimm**
will be needing to fix some image reducer part but no problem in using it

### CODE REVIEW

File: react-app/src/store/reviews.js [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+  payload: reviewImage
+})
+
+const removeReviewImage = (reviewImageId) => ({
+  type: REMOVE_REVIEW_IMAGE,
+  payload: reviewImageId
+})
+
+// Get all Reviews
+export const fetchReviews = () => async (dispatch) => {
```
Comment (wtorresjr) [inline react-app/src/store/reviews.js:46]: This one is missing try - catch blocks otherwise looks good will merge when its updated.

**Review by wtorresjr — CHANGES_REQUESTED** [review]
Add try catch blocks to fetch reviews thunk.

**Review by kryskimmel — APPROVED** [review]
All good!

**Review by kryskimmel — APPROVED** [review]
nice!

**Review by kryskimmel — APPROVED** [review]

**Review by kryskimmel — APPROVED** [review]

**Pull request merged**